sapphire/tests/transformations/test_celestial.py

```diff
@@ -159,5 +161,323 @@ def test_against_jpl_horizons(self):
         self.assertAlmostEqual(dec, dec_astropy, 2)
 
 
+def test_transformations():
```

Comment:
This 'test'/comparison/benchmark function should be in the scripts directory ( `/sapphire/scripts/...` )